frontend/src/app/applications/revision/revision-history.component.html

```diff
@@ -8,9 +8,17 @@ <h2 class="app-item-label">Revision history</h2>
     </tr>
   </thead>
   <tbody>
+    <tr *ngIf="application.type === 'noncommercial'">
+      <td>
+        <span class="usa-label tan">Submitted</span>
+      </td>
+      <td>{{application.createdAt | date: 'MM/dd/yyyy hh:mm a' }}</td>
+      <td>{{application.authEmail}}</td>
+    </tr>
     <tr *ngFor="let revision of application.revisions | SortArray: 'createdAt'">
       <td>
         <span class="usa-label" [ngClass]="{
+          'tan': revision.status === 'Submitted',
```

Comment:
How is the status of an application being submitted handled here.

Reply:
For a temp outfitter, when an application is initially created, the status is set to `Incomplete`. After the files finish uploading, the status is set to `Submitted` and a revision is created. Because noncommercial is set to `Submitted` on creation, no revision is created initially.